recipes/mike/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "mike" %}
+{% set version = "0.5.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mike-{{ version }}.tar.gz
+  sha256: eef9c6b146f20a18ff7e0512838fd57d4dc2f389d1c012fa2285c9642794aa85
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - mike = mike.driver:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - flake8
```

Comment:
Are `flake8` and `pypandoc` needed under `host`? I see that `pypandoc` is optionally needed to get the long description, but neither will cause `setup.py` to fail.